### PR TITLE
docs: fix jsdocs & README.md

### PR DESCRIPTION
**Pull request by Chilfish**
### 🔗 Linked issue

N/A

### ❓ Type of change

- [x] 📖 Documentation (updates to the documentation, readme, or JSdoc annotations)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

fix typos in README.md and jsdocs and the inverted arguments.

### 📝 Checklist

- [ ] I have linked an issue or discussion.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by Chilfish** [comment]
While reviewing the git diff, I noticed that some of the examples in the jsdocs use single quotes while others use double quotes. It might be a good idea to standardize them and use single quotes consistently.

File: src/render.ts [hunk 1]
```diff
@@ -113,12 +113,12 @@ export function codeBlock(
  *
  * ```js
  * md.table({
- *  columns: ["Breed", "Origin", "Size", "Temperament"],
+ *  columns: ['Breed', 'Origin', 'Size', 'Temperament'],
```
Comment (pi0) [inline src/render.ts:116]: We use prettier defaults for formatting and it used `""`

Comment (Chilfish) [reply src/render.ts]: Ok, I fixed them

**Review by uncenter — APPROVED** [review]

**Review by pi0 — APPROVED** [review]
Thanks ❤️

**Pull request merged**